pages/about.html

```diff
@@ -0,0 +1,126 @@
+                </ul>
+                <ul class="h-link">
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </article>
+        </nav>
+        <nav class="main-nav">
+            <section class="mobile-menu mb">
+                <label class="hamburger-menu mb" for="menu-checkbox">
+                    <input type="checkbox" id="menu-checkbox" />
+                </label>
+                <aside class="sidebar">
+                    <a class="new-stuff" href="#">Logout</a>
+                    <a href="../index.html">Home</a>
+                    <a href="#">Program</a>
+                    <a href="#">Join</a>
+                    <a href="#">Sponsor</a>
+                    <a href="#">News</a>
+                    <a class="CC" href="#">CC Campaign</a>
+                </aside>
+            </section>
+            <a href="../index.html" class="logo dk">BLK-NG</a>
+            <article class="main-nav-link dk">
+                <a href="../index.html">Home</a>
+                <a href="#">Program</a>
+                <a href="#">Join</a>
+                <a href="#">Sponsor</a>
+                <a href="#">News</a>
+                <a href="#">CC Campaign</a>
+            </article>
+        </nav>
+        <section class="headline-container-abt">
+            <section class="headline-abt">
+                <h2 class="grtings mb">"Hello Sharing world"</h2>
+                <h2 class="title-abt">
+                    CREATIVE COMMONS<br>GLOBAL SUMMIT 2023
+                </h2>
+                <p class="intro-txt-abt">
+                    The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                    This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.
+                    So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together
+                </p>
+                <p class="intro-txt-abt-1">
+                    Please contact us per Email for nay further questions about CC Global Summit 2023!
+                </p>
+                <a class="email intro-txt-abt-1" href="mailto:sagiecyber@gmail.com">sagiecyber@gmail.com</a>
+            </section>
+        </section>
+    </header>
+    <main>
+        <section class="main-abt">
+            <h2 class="title-abt-1">
+                Creative Commons Global Summit 2023 Logo
+            </h2>
```

Comment:
- [ ] 👍 Thank you for your efforts! As mentioned earlier, please adjust the color for this element, as it currently doesn't match the design. Kindly update your code to use the correct color as per the design. Keep up the good work! 🌟💻🚀